Update/omot_004.txt

```diff
@@ -1596,7 +1596,7 @@ void main()
 //rとりあえず梨花ちゃんのジャッジで、レナの攻撃は有効と承認。…しかしレナ、あいつは合気道でもやってるのか？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "とりあえず梨花ちゃんのジャッジで、レナの攻撃は有効と承認。…しかしレナ、あいつは合気道でもやってるのか？",
-			NULL, "For the time being, Rika-chan declared that Rena's attacks were both valid and approved. ...But since when could Rena use aikido?", Line_Normal);
+			NULL, "For the time being, Rika-chan declared that Rena's attacks were both valid and approved. ...But did Rena practice aikido or something?", Line_Normal);
```

Comment:
```suggestion
			NULL, "For the time being, Rika-chan declared that Rena's attacks were both valid and approved. ...But since when could Rena use aikido?", Line_Normal);
```